tasks/setup/InfracostSetup/index.ts

```diff
@@ -0,0 +1,284 @@
+import * as taskLib from 'azure-pipelines-task-lib/task';
+import * as toolLib from 'azure-pipelines-tool-lib/tool';
+import * as path from 'path';
+import * as os from 'os';
+import * as semver from 'semver';
+import fetch from 'node-fetch';
+
+/**
+ * archMappings defines how different https://nodejs.org/api/os.html#os_os_arch
+ * should be represented in terms of the infracost cli assets.
+ */
+const archMappings: { [s: string]: string } = {
+  x64: 'amd64',
+};
+
+/**
+ * mapArch returns a string of the correct binary arch to use in the pipeline.
+ *
+ * @param arch - the os architecture, arch in [arm, x32, x64...] (https://nodejs.org/api/os.html#os_os_arch)
+ */
+function mapArch(arch: string): string {
+  return archMappings[arch] || arch;
+}
+
+/**
+ * osMappings defines how different https://nodejs.org/api/os.html#os_os_platform
+ * should be represented in terms of the infracost cli assets.
```

Comment:
```suggestion
 * should be represented in terms of the Infracost CLI assets.
```

Reply:
👍

Reply:
done in https://github.com/infracost/infracost-azure-devops/pull/9/commits/95030bf9394c362cb56a6bdec90109b014fcd819